File: trunk/project/ponto/source/sistemas/admin/models/Menu.php
<?php
/**
 * Modelo da tabela menu
 *
 * @name Menu
 * @author Alexis Moura
 * @package _models
 * @version 1.0
 */
class Menu extends Axs_Db_Table
{
	/**
	 * Array de dados do modelo
	 *
	 * @name $_data
	 * @access public
	 * @var array
	 */
	public $_data = array(
		'ordemMenu' => null,
		'pathMenu' => null,
		'MenuIdPai' => null,
		'idMenu' => null,
		'nameMenu' => null,
		);
	/**
	 * Array de validação do modelo
	 *
	 * @name $_validators
	 * @access protected
	 * @var array
	 */
	protected $_validators = array(
		'ordemMenu' => array( 'Int','presence'=>'required'),
		'pathMenu' => array( 'notEmpty','presence'=>'required'),
		'MenuIdPai' => array( 'Int','presence'=>'required'),
		'idMenu' => array( 'allowEmpty'=>'true'),
		'nameMenu' => array( 'notEmpty','presence'=>'required'),
		);
	/**
	 * Array de descrição dos dados
	 *
	 * @name $descriptionFields
	 * @access public
	 * @var array
	 */
	public $descriptionFields = array(
		'ordemMenu' => 'Ordem',
		'pathMenu' => 'URL',
		'MenuIdPai' => 'Menu pai',
		'idMenu' => 'Código',
		'nameMenu' => 'Nome',
		);
	

}

?>

File: trunk/project/CSharp/ponto/PontoEncontro.Domain/Entidades/Menu.cs
using System;
using System.Collections.Generic;

namespace PontoEncontro.Domain
{
    [Serializable]
    public class Menu
    {


        public Menu()
        {
        }

        public Menu(int Idmenu, Menu Menuidpai, string Namemenu, int Ordemmenu, string Pathmenu)
        {
            this.Idmenu = Idmenu;
            this.Menuidpai = Menuidpai;
            this.Namemenu = Namemenu;
            this.Ordemmenu = Ordemmenu;
            this.Pathmenu = Pathmenu;
        }

        public virtual int Idmenu { get; set; }
        public virtual Menu Menuidpai { get; set; }
        public virtual string Namemenu { get; set; }
        public virtual int Ordemmenu { get; set; }
        public virtual string Pathmenu { get; set; }
        public virtual IList<Menu> MenuList { get; set; }
        public virtual IList<Menu> MenuidpaiList { get; set; }

    }
}
